samples/zigbee/shell/sample.yaml

```
@@ -2,12 +2,23 @@ sample:
   name: Zigbee application with Zigbee shell
   description: Zigbee application thast includes all Zigbee shell commands
 tests:
-  zigbee.template:
+  zigbee.shell:
     build_only: true
     platform_allow: nrf52840dk_nrf52840 nrf52833dk_nrf52833 nrf5340dk_nrf5340_cpuapp nrf21540dk_nrf52840
-    tags: ci_build smoke
+    tags: ci_build shell
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf52833dk_nrf52833
       - nrf5340dk_nrf5340_cpuapp
       - nrf21540dk_nrf52840
+
+  zigbee.shell_ed:
```

```suggestion
  zigbee.shell.ed:
```
Usually sample variant's name is separated using dots.